### ISSUE TITLE
Missing Geometry Fix Option for Most Wanted

### ISSUE THREAD

**Issue by AlexKVideos1** [AeroWidescreen/NFSHDReflections#5]
I remember in an older version of the HD Reflections for Most Wanted, there was an option called "GeometryFix" to fix the car garage preview and some object that were missing details. Aero showed off the issue in this [video](https://www.youtube.com/watch?v=tnJY4ziF2oA).

When I installed the newest version of the mod, I noticed this fix was no longer in the .ini file. While the car garage reflection fix is still included (I believe with RestoreBackfaceCulling), it seems the other fix for the missing object detail (like on the lamp posts), is not included anymore.
![Need for Speed  Most Wanted Old Screenshot 2023 02 02 - 23 24 48 01](https://user-images.githubusercontent.com/35385014/216512672-f253643e-c27e-4634-8514-b5066bee2cb6.jpg)

I figured I would report it because I think this might have gotten overlooked when the mod got updated.

**Comment by AeroWidescreen** [comment]
When I discovered this several years ago, I didn't understand that "GeometryFix" was actually disabling backface culling. It wasn't really fixing anything, just bypassing some modeling errors.

Here's the Xbox 360 release, often regarded as the graphically definitive version of Most Wanted, with the same problem:

![XB360](https://i.imgur.com/EMO7KPL.png)

As a result, I also thought this was the correct way to fix garage reflections. The problem was an error in how the game handled backface culling for inverted car models, so "GeometryFix" made them visible again. In reality, disabling backface culling actually caused more graphical issues, like z-fighting and windshield rendering bugs. `RestoreBackfaceCulling` was created as the proper solution for the broken garage reflections and "GeometryFix" was removed as a result.
